source/guide_ejabberd.rst

```diff
@@ -0,0 +1,503 @@
+    Länge: nicht spezifiziert [application/x-gzip]
+    In »»19.05.tar.gz«« speichern.
+
+        [    <=>                                                                              ] 1.865.845   1,51MB/s   in 1,2s
+
+    2019-07-10 09:48:55 (1,51 MB/s) - »19.05.tar.gz« gespeichert [1865845]
+
+    [isabell@stardust ~]$ tar xf 19.05.tar.gz
+    [isabell@stardust ~]$ cd ejabberd-19.05/
+    [isabell@stardust ejabberd-19.05]$./autogen.sh
+    [isabell@stardust ejabberd-19.05]$ ./configure --enable-user=$USER --prefix=$HOME/ejabberd
+    checking whether make sets $(MAKE)... yes
+    checking for a BSD-compatible install... /usr/bin/install -c
+    checking for a sed that does not truncate output... /usr/bin/sed
+    checking for erl... /home/isabell/erlang/builds/21.2/bin/erl
+    checking for erlc... /home/isabell/erlang/builds/21.2/bin/erlc
+    checking for epmd... /home/isabell/erlang/builds/21.2/bin/epmd
+    checking for erl... /home/isabell/erlang/builds/21.2/bin/erl
+    checking for erlc... /home/isabell/erlang/builds/21.2/bin/erlc
+    checking Erlang/OTP version... ok
+    checking for Erlang/OTP root directory... /home/isabell/erlang/builds/21.2
+    checking for escript... /home/isabell/erlang/builds/21.2/bin/escript
+    checking for make... make
+    allow this system user to start ejabberd: isabell
+    configure: creating ./config.status
+    config.status: creating Makefile
+    config.status: creating vars.config
+    config.status: creating src/ejabberd.app.src
+    [isabell@stardust ejabberd-19.05]$
+
+Install it:
+::
+
+    [isabell@stardust ejabberd-19.05]$ make install
+
+Make the controll script available:
+
+.. code-block:: console
+
+   [isabell@stardust ~]$ ln -s ~/ejabberd/sbin/ejabberdctl ~/bin/ejabberdctl
+   [isabell@stardust ~]$
+
+
+Configuration
+=============
+
+Open Firewall Ports
+-------------------
+ejabberd needs three open ports for c2s, s2s, and proxy connections. HTTP connections are handled by a web backend.
+
+.. include:: includes/open-port.rst
+
+As standard ports cannot be used on uberspace an external domain is needed and SRV records must be set for c2s and s2s connections. Refer to the `XMPP wiki <https://wiki.xmpp.org/web/SRV_Records>`_ for setup and point them to the corresponding ports.
+
+Change the configuration
+------------------------
+
+A standard config file is provided at ``~/ejabberd/etc/ejabberd/ejabberd.yaml`` where the ports, domain and administrator need to be set:
+
+Change the host configuration to listen for the correct domain:
+
+.. code-block:: ini
+ :emphasize-lines: 2
+
+  hosts:
+  - "isabell.example"
+
+Provide location of your keys and certificates for TLS transport:
+
+.. note:: The list can be expanded with certfiles for each domain and subdomain.
+.. code-block:: ini
+ :emphasize-lines: 2,3
+
+  certfiles:
+    - "/home/isabell/etc/certificates/isabell.example.crt"
+    - "/home/isabell/etc/certificates/isabell.example.key"
```

Comment:
I think there should be somehow written that you also need to include the certs for the subdomains